### PR TITLE
Address some of the issues identified by golint.

### PR DESCRIPTION
**Pull request by pedro-r-marques**
lint warnings that would result in modifying the API are not corrected
by this commit.

### CODE REVIEW

File: keystone.go [hunk 1]
```diff
@@ -6,35 +6,39 @@ package contrail
...
+	osTenantName string
+	osUsername   string
+	osPassword   string
+	osAdminToken string
 
 	current *KeystoneToken
 }
 
-// Token issued by keystone
+// KeystoneToken represents a auth token issued by OpenStack keystone service.
```
Comment (nati) [inline keystone.go]: an auth

Comment (pedro-r-marques) [inline keystone.go]: done

File: mocks/database.go [hunk 2]
```diff
@@ -290,15 +299,17 @@ func (db *InMemDatabase) Delete(obj contrail.IObject) error {
...
 	uid := makeUID(id)
-	obj, ok := db.objByIdMap[uid]
+	obj, ok := db.objByIDMap[uid]
 	if !ok {
 		return nil, fmt.Errorf("%s: Not found", id.String())
 	}
 	return obj, nil
 }
 
+// GetByName retrieves and object by colon separated fully qualified name.
```
Comment (nati) [inline mocks/database.go]: you mean "and makes object ? "

Comment (pedro-r-marques) [inline mocks/database.go]: It was supposed to read "an object". fixed.

File: mocks/uid.go [hunk 3]
```diff
@@ -4,13 +4,16 @@ import (
...
+// UID is defined since uuid.UUID is a slice and cannot be used as a map key.
 type UID [16]byte
 
+// Interface converts the fixed length array representation to the uuid package
+// type.
 func (u *UID) Interface() uuid.UUID {
 	return uuid.UUID(u[:])
 }
 
+// IsNIL returns true if the UID value is 0.
```
Comment (nati) [inline mocks/uid.go:16]: unrelated question: I can't understand this function.. it looks like returns always false.

File: mocks/updater.go [hunk 4]
```diff
@@ -11,12 +11,17 @@ type objectUpdater struct {
 	db Database
 }
 
+// NewObjectUpdater returns an object the implements the contrail.objectInterface interface.
+// These are the methods provoded to the generated types classes that allow them to manage
+// get,add,delete operations on references.
```
Comment (nati) [inline mocks/updater.go]: get, add and, delete operations ?

File: mocks/uid.go (hunk 3 above)
Comment (pedro-r-marques) [inline mocks/uid.go:16]: It returns true if all the 16 bytes are 0.

File: mocks/updater.go (hunk 4 above)
Comment (pedro-r-marques) [inline mocks/updater.go]: Done.

File: mocks/uid.go [hunk 5]
```diff
@@ -4,13 +4,16 @@ import (
...
 func (u *UID) IsNIL() bool {
 	for i := 0; i < 16; i++ {
 		if i != 0 {
```
Comment (nati) [inline mocks/uid.go]: then I think you should do u[i] != 0
or
more simply -> https://play.golang.org/p/7LA21ov9g6

Comment (pedro-r-marques) [inline mocks/uid.go]: Doh! Fixed.

**Comment by nati** [comment]
LGTM

**Pull request merged**